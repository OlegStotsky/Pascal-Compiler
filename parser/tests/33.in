procedure a (a : integer)
begin
a := 5;
end