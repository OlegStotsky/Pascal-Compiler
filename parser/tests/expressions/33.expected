Error at line 6, column 13 : can't cast type float to type integer
